give/src/components/Form/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+// Based on v2
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-form/form-button.html
+// https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/form/button/v2/button/button.html
+
+import { ComponentWrapper } from "../../shared";
+
+export const Button = ({
+  className = "", // string
+  type = "button", // string - "button" | "submit"
+  title = "", // string
+  name = "", // string
+  value = "", // string
+  id = "", // string
+}) => {
+  const types = ["button", "submit"];
+
+  if (!types.includes(type)) return null;
+
+  return (
+    <ComponentWrapper type="button" className={className} hideInner>
+      <button
+        type={type}
+        id={id ? id : null}
+        className="cmp-form-button"
+        name={name ? name : null}
```

Comment:
Why would null be a valid option? Wouldn't "" be instead or do you not want the attribute at all in that case? I would think a generic name like "cru-button" or something could be a default for at least the name

Reply:
I'm mimicking what AEM does and if the attributes aren't provided values, they aren't output at all. At least that's how our current AEM installation works. I don't know if this will change once we're on the cloud.